clang/runtime/dpct-rt/include/dpct/device.hpp

```diff
@@ -785,6 +785,34 @@ has_capability_or_fail(const sycl::device &dev,
     break;
   }
 }
+
+/// Util function to synchronize all the queues of current device with default
```

Comment:
/// Util function to insert a synchronize barrier in the queue and return the event of the barrier.